src/client_side.h

```diff
@@ -394,9 +394,11 @@ class ConnStateData : public Server, public HttpControlMsgSink, private Independ
 #if USE_OPENSSL
     bool switchedToHttps_;
     bool parsingTlsHandshake; ///< whether we are getting/parsing TLS Hello bytes
+    uint32_t parsedBumpedRequestCount; ///< The number of bumped HTTP requests
```

Comment:
Please use either 

* `unsigned int` (i.e. the simplest type that still works OK) if you do not care about overflows or
*  `uint64_t` (i.e., the type that works best) if you do care about overflows.

I recommend the latter.

Reply:
FYI: HTTP/1.x has no defined limit on pipeline lengths (it is only a tradition to use 5-10) and HTTP/2 and HTTP/3 have a 2^31 requirement which our Pipeline was being prepared for. Remember these are counters for accounting the entire potential week/month/year long lifetime of the connection.